action.yml

```diff
@@ -151,6 +156,21 @@ runs:
         fi
 
         pip install "${repo_url}/@${{inputs.nuitka-version }}#egg=nuitka"
+    - name: Install ccache
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        sudo apt-get install -y ccache
+        export PATH="/usr/lib/ccache:$PATH"
```

Comment:
I think the PATH modification ought to not be necessary for Nuitka, it looks for `ccache`  and executes that rather than say the 'gcc' link. But that's not too important, and of course a pattern to follow. But that PATH extension will not carry over to anything else anyway, right?

Reply:
After removing the path update, Nuitka still finds ccache, so it does seem unnecessary.

From a run after removing the update:
```bash
Nuitka-Scons:INFO: Found ccache '/usr/bin/ccache' to cache C compilation result.
```